test/integration/hotrestart_test.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+if [[ -f "test/integration/hotrestart_disabled" ]]; then
```

Comment:
See my comments on the use of test tags and interpretation in `envoy_build_system.bzl` as one possible option.